eman2/protocols/protocol_tomo_subtomogram_refinement.py

```diff
@@ -0,0 +1,243 @@
+# coding=utf-8
+# **************************************************************************
+# *
+# * Authors:     Adrian Quintana (adrian@eyeseetea.com) [1]
+# *              Ignacio del Cano  (idelcano@eyeseetea.com) [1]
+# *              Arnau Sanchez  (arnau@eyeseetea.com) [1]
+# *
+# * [1] EyeSeeTea Ltd, London, UK
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
+# * (at your option) any later version.
+# *
+# * This program is distributed in the hope that it will be useful,
+# * but WITHOUT ANY WARRANTY; without even the implied warranty of
+# * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# * GNU General Public License for more details.
+# *
+# * You should have received a copy of the GNU General Public License
+# * along with this program; if not, write to the Free Software
+# * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA
+# * 02111-1307  USA
+# *
+# *  All comments concerning this program package may be sent to the
+# *  e-mail address 'scipion@cnb.csic.es'
+# *
+# **************************************************************************
+from types import NoneType
+from glob import glob
+import os
+import re
+
+from pyworkflow import utils as pwutils
+import pyworkflow.protocol.params as params
+import pyworkflow.em as pwem
+from pyworkflow.em import Transform
```

Comment:
I think this import is unused

Reply:
Right https://github.com/scipion-em/scipion-em-eman2/pull/36/commits/df1ed7237ecdf2e47a930f08d0b5fe8ee516a487